mapswipe_workers/mapswipe_workers/utils/tasks_to_geojson.py

```diff
@@ -0,0 +1,109 @@
+from osgeo import ogr
+import os
+from mapswipe_workers.utils import tile_grouping_functions as t
+from mapswipe_workers.project_types.build_area.build_area_group import BuildAreaGroup
+from mapswipe_workers.project_types.build_area.build_area_task import BuildAreaTask
+from mapswipe_workers.project_types.build_area.build_area_project import (
+    BuildAreaProject,
+)
+
+
+from mapswipe_workers.definitions import logger
+
+url = (
```

Comment:
yes this URL needs to be defined, because I am trying to create tasks based on the geojson file. and in order to create tasks I need to feed the BuildAreaTask Class `task = BuildAreaTask(group, project, TileX, TileY)`, otherwise I get a tilserver missing error message